data/source/MongoDb.php

```diff
@@ -153,14 +181,14 @@ class MongoDb extends \lithium\data\Source {
 	 *
 	 * The tyes in the schema map to database native type like this:
 	 * ```
-	 *  id      => MongoId
-	 *  date    => MongoDate
-	 *  regex   => MongoRegex
+	 *  id      => MongoDB\BSON\ObjectId
+	 *  date    => MongoDB\BSON\UTCDateTime
+	 *  regex   => MongoDB\BSON\Regex
 	 *  integer => integer
 	 *  float   => float
 	 *  boolean => boolean
-	 *  code    => MongoCode
-	 *  binary  => MongoBinData
+	 *  code    => MongoDB\BSON\Javascript
+	 *  binary  => MongoDB\BSON\Binary
```

Comment:
Style cleanup: remove the extra whitespace between key and assignment operator.

Reply:
As this is just a comment, I'd leave it as it is, as there is focus on humans readability. What do you think?